### PR TITLE
add rdata from vignette so it is usable; generate in vignette-data

### PR DESCRIPTION
**Pull request by GitBrianLaw**
(@malecki doing this; brian to finish branch and notify when we can review)

### CODE REVIEW

File: vignettes/datasets.Rmd [hunk 1]
```diff
@@ -26,8 +26,10 @@ To create new datasets, multiple paths exist. In the web application, you can up
 
 You can create a dataset from any `data.frame` you have in your R session with `newDataset`:
 
-```{r, eval=FALSE}
-load("economist.RData")
+This package provides one as `economist` which loads a `data.frame` named `df` because old habits die hard.
```
Comment (nealrichardson) [inline vignettes/datasets.Rmd]: Why the snark? If there's a problem with the naming convention, please propose an alternative.

File: vignettes/datasets.Rmd [hunk 2]
```diff
@@ -26,8 +26,10 @@ To create new datasets, multiple paths exist. In the web application, you can up
...
+
+```{r}
+data("economist", package = "crunch") # named df
```
Comment (nealrichardson) [inline vignettes/datasets.Rmd]: I see below that vignettes/getting-started.RData is modified in this PR, but the file is generated from https://github.com/Crunch-io/rcrunch/blob/master/vignette-data/1-getting-started.R#L24, so any changes to it need to have a corresponding change in vignette-data/1-getting-started.R.

File: vignettes/datasets.Rmd (hunk 1 above)
Comment (malecki) [inline vignettes/datasets.Rmd]: Ha that was me. I do consider `df` not a great name, a result of longstanding old habits, and not great pedagogically. `economistExampleDf` or `economist` would be a better name here.

File: vignettes/datasets.Rmd (hunk 2 above)
Comment (malecki) [inline vignettes/datasets.Rmd]: @GitBrianLaw can add that. I thought we did. All of the issues can be resolved by renaming in `1-getting-started.R`

```
economist <- df
save(economist, file="../data/economist.RData")
```

Comment (nealrichardson) [inline vignettes/datasets.Rmd]: Almost. Also need to edit the line that follows this (here, in datasets.Rmd), which is `dim(df)`, as well as L44 of https://github.com/Crunch-io/rcrunch/blob/master/vignettes/variables.Rmd.

**Comment by nealrichardson** [comment]
Cleaning up our old issues. I'm going to close this and open a new issue (if we don't already have one) about using a public dataset in the vignettes so that you can play along at home.

**Pull request closed without merging**